### DISCUSSION TITLE
账号被封

### DISCUSSION THREAD

**Discussion by dsdffgh** [BurnHair/docs#147] [技术支持]
我没有重复注册过，之前的额度用完之每天签到攒额度，暂时没有用，突然发现被封了

**Comment by BennyThink** [comment]
<img width="223" alt="image" src="https://github.com/BurnHair/docs/assets/14024832/369510bf-2cf2-46db-9997-6f6b7114fd1f">

对我来说这些看起来很像是一个人

**Reply by dsdffgh** [reply]
啊，可是我确实只存在这一个账号，另外那些注册时间、签到时间应该也不会和我恰好一样吧？至于昵称随手打的呀。或许用了同一个机场，IP相似？

**Reply by BennyThink** [reply]
好的，已经解封了
